flask_advanced/blueprint/Products/utils.py

```diff
@@ -0,0 +1,13 @@
+import json
+
+
+def get_data():
+    with open("blueprint/Products/data.json") as file1:
```

Comment:
do not hardcode path to file, make this as param and your function will be universal